snuba/web/query.py

```diff
@@ -237,7 +239,10 @@ def _format_storage_query_and_run(
     visitor.visit(from_clause)
     table_names = ",".join(sorted(visitor.get_tables()))
     with sentry_sdk.start_span(description="create_query", op="db") as span:
-        formatted_query = format_query(clickhouse_query, request_settings)
+        _apply_turbo_sampling_if_needed(clickhouse_query, request_settings)
+
+        formatted_query = format_query(clickhouse_query)
+        formatted_query_anonymized = format_query_anonymized(clickhouse_query)
```

Comment:
you could wait to run `sql_anonymized` until `raw_query`, but i'm guessing it's better to pass the anonymized string via function param than to do any more logic in run_query

Reply:
Don't worry about that everything happens in the same thread, actually running the format in the raw query keeps the interface simpler so I would prefer that.